### ISSUE TITLE
Mylar causes login issues after a few days, reboot is only solution

### ISSUE THREAD

**Issue by DrSeussFreak** [SynoCommunity/spksrc#1701]
So recently I have had an issue on my DS412+ where it no longer allowed me to login with or without 2-factor authentication, it stopped responding via network shares and download manager ONLY works immediately after a reboot.

After troubleshooting with Synology, it was determined, with my satisfaction, that Mylar is the issue, and as soon as I uninstall it, all is well for all of the issues listed above, and even keeping it shutdown does not resolve the issue.

Hoping someone has some thoughts or troubleshooting steps I can take...

**Comment by DrSeussFreak** [comment]
So to add more info, uninstalling the mylar package and installing it via git, has resolved all my issues.  It seems that your package is somehow causing these issues.

**Comment by Diaoul** [comment]
Do you have other package repositories added to your NAS? Are you sure the Mylar package you're refering to is from SynoCommunity?

Mylar has the same codebase as other python packages and you're the first to report that issue. I doubt Mylar is really the issue, it seems local to your installation.

I'll leave the issue open for a few days to see if other people are facing the same problem.

**Comment by DrSeussFreak** [comment]
I do not have other repo's installed (I did, but no longer), and I was 100% positive that it was the SynoCommunity package.  

On my NAS I had performed 2 separate reset to factory settings and installed packaged back 1 at a time, and without a doubt Mylar was the issue. 

I can resolve the issue by uninstalling Mylar and only Mylar.

**Comment by Dr-Bean** [comment]
Fwiw, I've had Mylar installed for a day or so, currently running with 2-factor auth (usually I don't, but why not), and I haven't seen any issues.
Granted, Mylar isn't doing a whole lot, but it is running.

**Comment by DrSeussFreak** [comment]
What NAS do you have?  I have a DS413j and a DS412+, on the DS413j it is not a problem, but on the DS412+ is where I run into issues.

**Comment by Dr-Bean** [comment]
Well, that's just weird.
I'm currently running Mylar on a DS3615xs, so that's Bromolow. DS412+ is Cedarview, and those two  actually share the same Python package.
The Mylar package isn't even arch specific: all arches share the same package.

So if it were a general issue with the Python package, then it seems strange that it would only affect Mylar and not similar packages such as Sickbeard, CouchPotato etc.
On the other hand, if it were a general issue with Mylar, then it's strange that you're the only affected person. I do assume that you were running a stock/unmodified Mylar package here.

From the info here, it would seem more likely there's a third factor. Something which seems local to your setup, maybe another package or a file- or configuration change somewhere. It could be anything at this point, there's simply not enough info to determine that.
Maybe Synology has some info on how they found that Mylar was the issue?

**Comment by DrSeussFreak** [comment]
Synology only said they could not find any issues within the synology OS, and asked me to remove 3rd party apps.   

You are correct that I was running an unmodified Mylar package from your repository, as well as the Python package it required.

If I were to install it now the issues would come back (I tried this), yet when I installed it directly via Github all is well.  So not sure what is going on, but a fresh DS412+ without any customizations except the SynoCommunity packages installed have this issue, so it seems unlikely to a specific configuration of mine.
